resources/lang/en/events/misc/desert.json

```
@@ -0,0 +1,46 @@
+[
+  {
+    "event_id": "dst_misc_any_angrylizard",
+    "location": ["desert"],
+    "season": ["newleaf", "greenleaf", "leaf-fall"],
+    "weight": 20,
+    "event_text": "m_c was sniffing at an odd lump in the sand when suddenly an upset lizard gave {PRONOUN/m_c/object} a nip on the nose and dashed away!",
```

```suggestion
    "event_text": "m_c was nipped on the nose when {PRONOUN/m_c/subject} investigated an odd, lizard-shaped lump in the sand.",
```
This does imply an injury occurred, but it's marked as misc?

---

I'm not 100% sure if it should be? I have a pet gecko myself and whilst yes their bites HURT they rarely cause damage,  I could move it into injuries as a small cut though since it could be a more fierce lizard